apps/veil/src/pages/tournament/server/delegator-history.ts

```diff
@@ -32,162 +31,118 @@ export interface TournamentDelegatorHistoryResponse {
 export interface LqtDelegatorHistoryData extends Omit<LqtDelegatorHistory, 'address' | 'asset_id'> {
   address: Address;
   asset_id: AssetId;
-  metadata: Metadata;
 }
 
-const tournamentDelegatorHistoryQuery = async ({ epoch }: { epoch: number }) => {
-  return pindexerDb
-    .selectFrom('lqt.delegator_history')
-    .orderBy('epoch', 'desc')
-    .orderBy('power', 'desc')
-    .where('epoch', '=', epoch)
-    .selectAll()
-    .execute();
-};
+export const getBodyParams = async (
+  req: NextRequest,
+): Promise<Required<DelegatorHistoryRequest> | string> => {
+  const body = (await req.json()) as DelegatorHistoryRequest;
 
-/**
- * Consolidate per-epoch query results into a single history for the given
- * delegator address.
- *
- * For each epoch we:
- *    * filter rows belonging to the delegator,
- *    * tally their total voting power,
- *    * capture that epoch’s reward and asset id.
- *
- * Finally, we aggregate the rewards earned by the address across all epochs.
- */
-function processEpochResults(
```

Comment:
removed this function completely in favor of vanilla SQL calculations

Reply:
love this!